pkg/nativetest/common_test.go

```diff
@@ -0,0 +1,102 @@
+package nativetest
```

Comment:
It is, but we can't place it directly to `pkg/core/native` because of import cycle problem. `nativetest` uses `neotest` and `neotest` uses core. I moved `nativetest` package to `pkg/core/native` to keep things ordered.

Reply:
`package native_test`?